player/tests.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import get_user_model
+from .models import Player, BallGame
+import pytest
+
+
+@pytest.fixture
+def user():
+    return get_user_model().objects.create_user(
+        username='testuser',
+        password='testpass'
+    )
+
+
+@pytest.fixture
+def player(user):
+    return Player.objects.create(
+        user=user,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+
+
+def test_create_player(player, user):
```

Comment:
I'd expect to have a function that will encapsulate the creation of the user when creating the player. If this is something that you'd prefer doing later due to time limitations, then you can create an issue for it and do it after you're ready for the demo.

Reply:
Can you create an issue for that? I think this sort of method will be needed later on, but we can implement it when needed. I don't want it to hold this PR at this point, but I don't want it to be forgotten. You can resolve the conversation once you open the issue.

Reply:
ok, but what do you mean by encapsulating the user cration ? 
can you give me basic example maybe ?

Reply:
I'd think that you wouldn't have to create the user explicitly and that the creation of the player will also create the user. Need to make sure that is indeed a good idea and how to do it, so I don't want to do it now. We'll revisit it later when we have some more time.